### DISCUSSION TITLE
ERROR settings validation: missing port in address

### DISCUSSION THREAD

**Discussion by bmpreston** [qdm12/ddns-updater#483] [Q&A] [answered]
I had updater working until the other day, log shows the attached image and this text:

ERROR settings validation: health settings: server listening address: splitting host and port: address 10s: missing port in address

Config is also attached and nothing has recently KNOWINGLY changed.  Thanks!

<img width="1180" alt="Screenshot 2023-06-12 at 20 14 55" src="https://github.com/qdm12/ddns-updater/assets/29456939/842eee25-395e-48c6-8bb7-2b76ea9fe942">

<img width="579" alt="Screenshot 2023-06-12 at 20 13 01" src="https://github.com/qdm12/ddns-updater/assets/29456939/9c42d45c-da2e-4ecb-b23c-7076c2209b82">

**Comment by qdm12** [comment, marked as answer]
That was a bug in the latest image introduced in 4b51c8b35b943433b7adf7d60a74f06e475d97b8 (see the announcement in logs 

> Environment variables parsing was changed on 12 June, please report any issue you might have

This got fixed yesterday as well with 3e6fb5ead44488ed2deb44fc08ef91915513dc45

I think you were unlucky enough to pull the borked latest image yesterday morning, pulling it again will resolve it 😉
